include/oneapi/dpl/pstl/parallel_impl.h

```diff
@@ -79,16 +82,18 @@ __parallel_or(__parallel_tag<_IsVector> __tag, _ExecutionPolicy&& __exec, _Index
 {
     using __backend_tag = typename __parallel_tag<_IsVector>::__backend_tag;
 
-    ::std::atomic<bool> __found(false);
-    __par_backend::__parallel_for(__backend_tag{}, ::std::forward<_ExecutionPolicy>(__exec), __first, __last,
-                                  [__f, &__found](_Index __i, _Index __j) {
-                                      if (!__found.load(::std::memory_order_relaxed) && __f(__i, __j))
-                                      {
-                                          __found.store(true, ::std::memory_order_relaxed);
-                                          __par_backend::__cancel_execution(__backend_tag{});
-                                      }
-                                  });
-    return __found;
+    return __internal::__except_handler([&]() {
+        std::atomic<bool> __found(false);
+        __par_backend::__parallel_for(__backend_tag{}, std::forward<_ExecutionPolicy>(__exec), __first, __last,
```

Comment:
This change isn't required because `__internal::__except_handler` used on caller side of `__parallel_or` function.